packages/webpack-cli/lib/webpack-cli.js

```diff
@@ -213,6 +215,30 @@ class WebpackCLI extends GroupHelper {
         this._mergeOptionsToConfiguration(options);
     }
 
+    /**
+     * Responsible for handling flags coming from webpack/webpack
+     * @private\
+     * @returns {void}
+     */
+    _handleCoreFlags() {
+        if (this.groupMap.has('core')) {
+            const coreFlags = this.groupMap.get('core');
+
+            // convert all the flags from map to single object
+            const coreConfig = coreFlags.reduce((allFlag, curFlag) => (allFlag = { ...allFlag, ...curFlag }), {});
+
+            // for some reason these flags are always appended now, remove this later
+            delete coreConfig['module-no-parse'];
```

Comment:
Can you also investigate why these properties are added when importing core flags?

Reply:
looking into it.

Reply:
@anshumanv corresponding flags are broken as well :/